multiple-matches.suite/Scripts/spitta.script

```diff
@@ -0,0 +1,9 @@
+﻿
+
+Put EveryImageLocation("spitta") into FoundImages
+
+// Use the next to line to find the one you want. Then you can comment it out if you like.
+// log EveryImageLocation("spitta")
+
+Click item 4 of FoundImages
```

Comment:
Rather than clicking a specific item in the array, there should be some way to identify exactly which label you're targeting and clicking the corresponding ellipsis button. If you check the automation requirements for this challenge, you need to make it extensible for all labels, but specifically verify it's finding the appropriate one for the **EXTR** field.